aioftp/client.py

```diff
@@ -137,6 +139,8 @@ def __init__(self,
         self.path_io = path_io_factory(timeout=path_timeout, loop=loop)
 
         self.stream = None
+        # 2: use MLSD,1:use LIST,0:use NLST/CWD(not implemented)
+        self.cwd = None
```

Comment:
No need to cash cwd and all this stuff around. Left it as is, since `CWD` is really not using.